flink-connector/src/main/java/org/apache/flink/connector/delta/sink/writer/DeltaWriter.java

```diff
@@ -63,6 +64,20 @@
 
```

Comment:
Can we rename this class tot DeltaSinkWriter? i think that makes it more consistent with the pattern other classes follow: DeltaX extends X